README.md

```diff
@@ -243,58 +313,43 @@ dokkaHtml {
     }
 ```
 
-For convenience, there is also a reserved block called `global`, which is a top-level configuration of `perPackageOptions`, `externalDocumentationLinks`, and `sourceLinks` shared by every platform. Eg.
+If you want to share the configuration between source sets, you can use Gradle's `configureEach`
 
-```groovy
-dokkaHtml {
-    multiplatform {
-        global { // perPackageOptions, sourceLinks and externalDocumentationLinks from here will be copied to every other platform (jvm and js in eg.)
-            perPackageOption {
-                prefix = "com.somePackage"
-                suppress = true
-            }
-            perPackageOption { // You can repeat this block for multiple perPackageOptions
-                prefix = "kotlin" 
-                skipDeprecated = false
-                reportUndocumented = true
-                includeNonPublic = false
-            }
-            sourceLink {
-                path = "src/main/kotlin" 
-                url = "https://github.com/cy6erGn0m/vertx3-lang-kotlin/blob/master/src/main/kotlin" 
-                lineSuffix = "#L"
-            }
-            externalDocumentationLink {
-                url = new URL("https://example.com/docs/")
-            }
-       }
-       js {}
-       jvm {}
+#### Applying plugins
+Dokka plugin creates Gradle configuration for each output format in the form of `dokka${format}Plugin`:
+
+```kotlin
+dependencies {
+    dokkaHtmlPlugin("org.jetbrains.dokka:kotlin-as-java-plugin:1.4-M3")
+}
+``` 
+
+You can also create a custom dokka task and add plugins directly inside:
+
+```kotlin
+val customDokkaTask by creating(DokkaTask::class) {
+    dependencies {
+        plugins("org.jetbrains.dokka:kotlin-as-java-plugin:1.4-M3")
```

Comment:
Side note: Should we consider calling this porperty `plugin`? The plural looks strange here!

Reply:
We should, I agree